src/app.js

```diff
@@ -610,6 +611,10 @@ var run = function () {
     }
 
     loadVersion(selectedVersion)
+
+    // static analysis
+    var staticanalysis = new StaticAnalysis(compiler, renderer)
```

Comment:
Why do you add it here? It should be rather similar to the "Formal Verification" tab, which is added right at the beginning.